main.cpp

```diff
@@ -52,106 +52,115 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: lion
+//  action 1: the lion eats prey
+lion.eatPrey();
+//  action 2: the lion sleeps in a cave
+lion.sleepInCave();
+//  action 3: the lion feeds her children
+lion.feedChildren();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: dolphin
+//  action 1: the dolphin jumps out of water
+lion.jumpOutWater();
+//  action 2: the dolphin fights a shark
+dolphin.fightShark();
+//  action 3: the dolphin becomes sentient
+dolphin.discoverReality();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Brooklyn
+//  action 1: Brooklyn snows
+brooklyn.heavySnow();

```

Comment:
Brooklyn doesn't snow.  
The weather snows in brooklyn. 
Also, "heavy snow" is a noun, not a verb. 
Pick a different action.
- [ ] revise